config/routes.rb

```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
   root "welcome#index"
+
+  resources :foods
```

Comment:
We would rather see you only create the routes that you will use. This should be `resources :foods, only: [:index]`